static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -575,15 +598,18 @@ export const getEpicTestsFromGoogleDoc = (): Promise<
                         ? rowWithUserCohort.userCohort
                         : 'OnlyNonSupporters';
 
-                    // If hasCountryName is true but a country name is not available for this user then
+                    // If testHasCountryName is true but a country name is not available for this user then
                     // they will be excluded from this test
-                    const hasCountryName = rows.some(row =>
+                    const testHasCountryName = rows.some(row =>
                         optionalStringToBoolean(row.hasCountryName)
                     );
 
+                    const geolocation = geolocationGetSync();
```

Comment:
Could geolocation be a global variable?

Reply:
good question. It's not possible because the hardcoded tests (e.g. https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-country-name.js) need access to the geolocation

Reply:
It needs to exist for the banner and epic separately because of the hard-coded tests